packages/validation/src/Questionary/date.ts

```diff
@@ -29,10 +29,14 @@ export const dateQuestionValidationSchema = (field: any) => {
 
   if (config.required) {
     schema = Yup.date()
+      .nullable()
```

Comment:
I wonder if it is required, why would it be null?

Reply:
I think so l used this documentation https://www.npmjs.com/package/yup 
`Mark the schema as required, which will not allow undefined or null as a value. Note that unless a schema is marked as nullable() a null value is treated as a type error, not a missing value`